src/app/components/harvest/screens/metadata-review/metadata-review.component.html

```diff
@@ -75,33 +75,51 @@ <h3>Review</h3>
 </div>
 
 <div class="mt-3 d-flex justify-content-between">
-  <button
+  <div>
+    <button
+      id="harvest-abort-button"
```

Comment:
is this id needed?

Reply:
It's used in the unit tests, but not in page functionality

Reply:
that is an excellent reason for it not to be there

ids are gloabal identifiers attached to a document

they should not be repeated and should not be used unless needed